moto/mediastoredata/urls.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import unicode_literals
+from .responses import MediaStoreDataResponse
+
+url_bases = [
+    "https?://data.mediastore.(.+).amazonaws.com",
+]
+
+response = MediaStoreDataResponse()
+
+url_paths = {
+    "{0}/$": response.dispatch,
```

Comment:
The `delete_object` calls has this signature:
`DELETE http://data.mediastore.eu-west-1.amazonaws.com/foo`

To intercept this call, the url-path has to match this pattern:
`"{0}/(?P<Path>[^/.]+)$": response.dispatch`

Note the `<Path>`-identifier, which allows us to retrieve this variable again in the Response-class:
`item_id = self._get_param("Path")`

Reply:
The `mediapackage`-service has more examples on intercepting URLs like this:
https://github.com/spulec/moto/blob/master/moto/mediapackage/urls.py

Reply:
I think it's ok now.